server/sonar-server/src/test/java/org/sonar/server/qualitygate/QualityGateFinderTest.java

```diff
@@ -39,16 +38,16 @@
   public ExpectedException expectedException = ExpectedException.none();
 
   @Rule
-  public DbTester dbTester = DbTester.create(System2.INSTANCE);
+  public DbTester db = DbTester.create(System2.INSTANCE);
```

Comment:
Hum, it's a convention that we're using with Teryk. Indeed, we should all use the same one !
For us, it's "db" because it's the root of everything related to db.
Then come dbClient, dbSession, etc.
To be discussed